package.json

```diff
@@ -168,14 +168,14 @@
     "react-banner": "^1.0.0-rc.0",
     "react-document-title": "^2.0.3",
     "react-dom": "^16.8.6",
-    "react-g-analytics": "0.4.2",
     "react-hot-loader": "^4.0.0-beta.12",
     "react-markdown": "4.2.2",
+    "react-plausible-analytics": "git+https://codeberg.org/resynth1943/react-plausible-analytics.git#master",
```

Comment:
Should be published to npm

Reply:
It really shouldn't.

I refuse to support the centralisation of worldwide software development, which is being [performed](https://news.microsoft.com/2018/06/04/microsoft-to-acquire-github-for-7-5-billion/) by Microsoft. I do not believe supporting them by uploading **my** code is ethical, as I stand in solidarity with the FOSS community. 

Furthermore, I refuse to support the cornering of FOSS development being done by Microsoft. [I know what they are capable of](https://en.wikipedia.org/wiki/Embrace%2C_extend_and_extinguish).

That being said, this may sound slightly hypocritical from someone who uses GitHub. Please remember that my only aim on this unhealthy platform is to relieve FOSS projects and their users of being burdened by monolithic, privacy-abusing nonfree software that is a risk to everyone's privacy, and *will never aim to serve the user.*

That's why I refuse to support the NPM ecosystem by lodging **my** source code on their registry. They are a ["cancer"](https://www.zdnet.com/article/microsoft-once-called-linux-a-cancer-and-that-was-a-big-mistake/) to software development.

Reply:
Hi @sokra, is this review resolved?

Reply:
@evilebottnawi This review has been concluded (or so I thought). I believe the issue has been addressed, but has not been marked as such. Could you please shine some light on this?